apiary-metastore-events/kafka-metastore-events/kafka-metastore-receiver/README.md

```diff
@@ -0,0 +1,16 @@
+# Apiary Kafka Metastore Receiver
+
+## Overview
+
+This module contains classes for retrieving and processing Hive Metastore events from Kafka.
+
+## Kafa Metastore Receiver Configuration
+
+To configure the receiver, a builder is provided. The builder requires that bootstrap servers, a topic name and a group id are specified. 
```

Comment:
I had left a comment earlier about providing a sample code here that can be copy-pasted to get started on using this receiver. You think that would be a good addition here?